internal/publish/publish_test.go

```diff
@@ -129,13 +130,40 @@ func issueJWT(t *testing.T, cfg jwtConfig) (jwtText, hmacKey string) {
 	return
 }
 
+type nameAssigner struct {
+	baseAPK      string
+	modifier     int
+	lastAssigned string
+}
+
+func makeNameAssigner(base string) *nameAssigner {
+	return &nameAssigner{
+		baseAPK: base,
+	}
+}
+
+func (n *nameAssigner) next() string {
+	n.modifier++
+	n.lastAssigned = fmt.Sprintf("%s.%d", n.baseAPK, n.modifier)
+	return n.lastAssigned
+}
+
+func (n *nameAssigner) last() string {
```

Comment:
I might call it "peek()" or "current()" given that it's the only one kept around.

Reply:
renamed to current